docs/app/partials/getting-started.tmpl.html

```diff
@@ -1,38 +1,84 @@
 <div ng-controller="GuideCtrl" class="doc-content">
   <md-content>
-    <p><em>New to Angular.js? Before getting into Angular Material, it might be helpful to
-      <a href="https://egghead.io/articles/new-to-angularjs-start-learning-here" target="_blank"
-         title="Link opens in a new window">read about the framework</a>.</em></p>
+    <p><em>New to Angular.js? Before getting into Angular Material, it might be helpful to:</em>
+    <ul>
+      <li>watch the videos about <a href="https://egghead.io/articles/new-to-angularjs-start-learning-here" target="_blank"
+               title="Link opens in a new window">Angular.js framework</a></li>
```

Comment:
My interpretation is that when the user hovers over the link, it will show this tip of "intended action".